src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -652,6 +652,11 @@ private bool InferConnectionDetails()
                 }
 
                 Port = port;
+
```

Comment:
Please include a comment..
`// If user is connecting to a managed instance with port, e.g. localhost\INSTANCENAME,1434`

Reply:
Suggested comment was added.